utils/src/test/java/DummyObject.java

```diff
@@ -0,0 +1,34 @@
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class DummyObject {
+    //class used only for Json testing
```

Comment:
Dangling comment

Reply:
I've replaced this ugly comment with JavaDoc comment.
Not every mock has to be created for example by Mockito. It'a allowed sometimes to use fake classes (look at Chapter 3 in "Working Effectively with Legacy Code" by Michael Feathers).